README.md

```diff
@@ -36,56 +36,68 @@ concept Person identified by name  {
 🔎 Introspect the model using a [powerful set of APIs](https://docs.accordproject.org/docs/model-api.html)
 
 🎛 Convert the model to other formats:
-- JSON Schema
-- XML Schema
-- OData CDSL
-- GraphQL Schema
-- Java Classes
-- Go Types
-- C# Classes
-- TypeScript Classes
-- Protobuf Messages
-- PlantUML Diagrams
-- Mermaid UML Diagrams
-- Markdown (with embedded Mermaid)
-- OpenAPI v3 specification document
-- Apache Avro
+
+-   JSON Schema
+-   XML Schema
+-   OData CDSL
+-   GraphQL Schema
+-   Java Classes
+-   Go Types
+-   C# Classes
+-   TypeScript Classes
+-   Protobuf Messages
+-   PlantUML Diagrams
+-   Mermaid UML Diagrams
+-   Markdown (with embedded Mermaid)
+-   OpenAPI v3 specification document
+-   Apache Avro
 
 🕸 Publish your reusable models to any website, including the Accord Project [model repository](https://models.accordproject.org)
 
 Infer models from other formats:
-- JSON document
-- JSON Schema
-- OpenAPI v3 specification document
+
+-   JSON document
+-   JSON Schema
+-   OpenAPI v3 specification document
+
+## Features
+
+-   **Expressive Model Language**: Capture business models succinctly.
+-   **Serialization**: Convert instances to JSON.
+-   **Deserialization and Validation**: Validate instances from JSON.
+-   **Model Introspection**: Utilize a powerful set of APIs.
+-   **Format Conversion**: Convert models to various formats such as JSON Schema, GraphQL Schema, Java Classes, and more.
+-   **Model Publication**: Publish reusable models to websites or repositories.
 
 ## Getting Started
 
-- Install the [Command Line Tool](https://concerto.accordproject.org/docs/tools/ref-concerto-cli)
-- Read the [Concerto specification](https://docs.accordproject.org/docs/model-concerto.html)
+-   Install the [Command Line Tool](https://concerto.accordproject.org/docs/tools/ref-concerto-cli)
+-   Read the [Concerto specification](https://docs.accordproject.org/docs/model-concerto.html)
 
 ```console
 $ npm i -g @accordproject/concerto-cli
-$ concerto compare --old model.cto --new model-with-changes.cto 
-[required-field-added]: The required field "weight" was added to the concept "Person" (major) 
+$ concerto compare --old model.cto --new model-with-changes.cto
+[required-field-added]: The required field "weight" was added to the concept "Person" (major)
 ```
 
-- Open VSCode ([on the web](https://github.dev/accordproject/models/blob/master/src/address%400.2.0.cto), [on your machine](https://marketplace.visualstudio.com/items?itemName=accordproject.cicero-vscode-extension))
+-   Open VSCode ([on the web](https://github.dev/accordproject/models/blob/master/src/address%400.2.0.cto), [on your machine](https://marketplace.visualstudio.com/items?itemName=accordproject.cicero-vscode-extension))
```

Comment:
Can youn please update the marketplace link here? We have a separate extension for concerto now.
```suggestion
-   Open VSCode ([on the web](https://github.dev/accordproject/models/blob/master/src/address%400.2.0.cto), [on your machine](https://marketplace.visualstudio.com/items?itemName=accordproject.concerto-vscode-extension))
```